pkg/controllers/webhook/utils.go

```diff
@@ -31,6 +31,9 @@ func (wh *webhook) buildRulesWithOperations(ops ...admissionregistrationv1.Opera
 	var rules []admissionregistrationv1.RuleWithOperations
 	for gvr := range wh.rules {
 		resources := sets.New(gvr.Resource)
+		if resources.Has("pods") {
```

Comment:
Shouldn't this be handled in https://github.com/kyverno/kyverno/blob/c5ab5a0fd31a6ee8e98187c64c63ea6c0f4a27cd/pkg/controllers/webhook/controller.go#L815-L868 ?

Reply:
Does this avoid adding duplicate entries for `pod/ephemeralcontainers`?

Reply:
> Shouldn't this be handled in 

It was moved from there in https://github.com/kyverno/kyverno/pull/4986

Reply:
> Does this avoid adding duplicate entries for pod/ephemeralcontainers?

We build webhook from scratch while updating it, it won't create duplicate entries.

Reply:
> Does this avoid adding duplicate entries for pod/ephemeralcontainers?

Yes, it does.